### ISSUE TITLE
bug: deaths with no killer (assist only) do not appear in the death list

### ISSUE THREAD

**Issue by patters-code** [TibiaData/tibiadata-api-go#203]
### Problem
Not all kills shown on tibia.com appear in the deathlog in tibiadata

### Expected behavior
all kills are shown, even if some fields cannot be populated

### Detail
Any death without any identifiable killer that only has "assists" is omitted from the death list. These deaths can occur when for example a character is killed only by members of their own guild. Presumably these kills are omitted because not all information is available, (Level the death occurred at etc). 

I think these deaths should still appear in the deathlist for the character, otherwise this deathlist does not accurately reflect what happened.

### Examples
Characters which (currently) serve as examples:
- [Sir Pixxy](https://www.tibia.com/community/?name=Sir+Pixxy)
- [Suedi Baaz](https://www.tibia.com/community/?name=Suedi+Baaz)

### Extra Background
Deaths of this kind are common on retro-hardcore servers where kill selling is widespread, as being in the same guild as the kill seller reduces protection zone block so buyers do not have to wait the full 15 minutes.

**Comment by tobiasehlert** [comment]
I'm looking into this and have stored away the required html to make a fix for this.

The interesting thing with for instance [Sir Pixxy](https://www.tibia.com/community/?name=Sir+Pixxy) is that the level is not present in the death entry. Thats why the regex is not catching that entry in the first place.

**Comment by pattersuk** [comment]
yeah - it seems like as it can't be logged as "died at level X by" because it was not classed as a PvE death, and it can't be logged as "killed at level X by" because after they remove "Assist" players from the list of killers, there are none, so it seems it just skips the "cause" part of the log (which is what includes the level) and prints the assists.

You can see this causes similar problems for other fansites too e.g. here: [https://www.tibiaring.com/char.php?c=Sir+Pixxy](https://www.tibiaring.com/char.php?c=Sir+Pixxy) it shows multiple deaths all at the same level because the level is omitted in this situation.

In an ideal world I'd love for Cip to change this behaviour so that the level the character dies at is printed, and it's clarified that it was a PvP death e.g. if the killers list is empty just write something vague "Killed at level X in player combat, Assisted by: Y". I feel like these two data points are "required fields" from the players perspective that should always be conveyed (somehow) in the death log. That's a matter for the proposals forum though!

**Comment by Pvmendes** [comment]
@tobiasehlert  this bug is still available?
